src/tools/annotation/bidirectionalTool/moveHandle/moveLongLine/updatePerpendicularLine.js

```diff
@@ -0,0 +1,64 @@
+import getLineVector from '../../utils/getLineVector';
+import getDistanceWithPixelSpacing from '../../utils/getDistanceWithPixelSpacing';
+
+/**
+ * Returns the updated line object that will be used to change the position of
+ * the perpendicular line handles.
+ *
+ * @param {*} baseData Base data for bidirectional line moving
+ * @param {*} mid Middle point considering the proposed point
+ *
+ * @returns {*} Returns a line object with the updated handles position
+ */
+export default function updatePerpendicularLine(baseData, mid) {
+  const {
+    columnPixelSpacing,
+    rowPixelSpacing,
+    start,
+    perpendicularStart,
+    perpendicularEnd,
+    intersection,
+    fixedPoint,
+  } = baseData;
```

Comment:
This does use more/most of `baseData` than other methods. I'm not opposed to passing the entire object if it makes your life easier, but having a JSDoc definition for the object would add some clarity. If we'd rather not maintain the `@typdef`, then destructuring in the signature might be a preferable alternative.